api/ordinals/package.json

```
@@ -0,0 +1,74 @@
+{
+  "name": "@hirosystems/ordinals-api",
+  "description": "A microservice that indexes Bitcoin Ordinal inscription data and exposes it via REST API endpoints.",
+  "version": "1.0.0",
+  "main": "index.js",
+  "author": "Hiro Systems PBC <engineering@hiro.so> (https://hiro.so)",
+  "license": "Apache 2.0",
+  "scripts": {
+    "build": "rimraf ./dist && tsc --project tsconfig.build.json",
+    "start": "node dist/src/index.js",
+    "start-ts": "ts-node ./src/index.ts",
+    "test": "jest --runInBand",
+    "test:brc-20": "npm run test -- ./tests/brc-20/",
+    "test:api": "npm run test -- ./tests/api/",
+    "lint:eslint": "eslint . --ext .js,.jsx,.ts,.tsx -f unix",
+    "lint:prettier": "prettier --check src/**/*.ts tests/**/*.ts",
```

I’ve noticed in the [.eslintrc.js ignorePatterns](https://github.com/hirosystems/ordhook/blob/da3bd057976ee04ee87db0c04d72669c531539ec/api/ordinals/.eslintrc.js#L12) that the client folder is ignored along with `.js` files in the tests folder. However, there are no `.js` files in tests.

Is there a specific reason why the `client` folder should not be formatted? Currently, Prettier does not check the client folder, and there was no command to explicitly run Prettier on the checked files. I’ve added it here, but please let me know if this change should not be made.

```suggestion
    "lint:prettier": "prettier --check client/**/*.ts src/**/*.ts tests/**/*.ts",
    "lint:prettier:fix": "prettier --write client/**/*.ts src/**/*.ts tests/**/*.ts",
```

---

Client libs are auto-generated most of the time, so we don't enforce our style rules there. I think we should keep it as is (we follow this standard in all APIs)